drivers/winc1500/winc1500_netdev.c

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright (C) 2017 Bumsik Kim <k.bumsik@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     driver_winc1500
+ * @{
+ *
+ * @file
+ * @brief       Netdev port for the WINC1500 driver
+ *
+ * @author      Bumsik Kim <k.bumsik@gmail.com>
+ *
+ * @}
+ */
+
+#ifdef MODULE_GNRC_NETDEV
```

Comment:
This looks more like `netdev` specific-code. `gnrc_netdev` is just the glue-code between GNRC and `netdev`. `netdev` can be used with other network stacks (like `lwIP`).

Reply:
Do you mean using `MODULE_NETDEV_ETH` instead? I am a bit confused :sweat_smile:

Reply:
Since without netdev this functions won't be used, optimization throws them out anyway, so no guard is needed.

Reply:
(but yes, *if* a guard is needed (e.g. for struct members), then `MODULE_NETDEV_ETH` would be better).

Reply:
After tried. It needs the guard because of a lot of NETDEV specific struct members in the code. So I will keep the guard.